libvast/vast/system/datagram_source.hpp

```diff
@@ -10,38 +10,18 @@
 
 #include "vast/fwd.hpp"
 
-#include "vast/concept/printable/std/chrono.hpp"
-#include "vast/concept/printable/stream.hpp"
-#include "vast/concept/printable/vast/error.hpp"
-#include "vast/concept/printable/vast/expression.hpp"
-#include "vast/defaults.hpp"
-#include "vast/detail/assert.hpp"
-#include "vast/error.hpp"
-#include "vast/expression.hpp"
-#include "vast/expression_visitors.hpp"
-#include "vast/logger.hpp"
-#include "vast/schema.hpp"
+#include "vast/system/actors.hpp"
 #include "vast/system/source.hpp"
-#include "vast/table_slice.hpp"
-#include "vast/table_slice_builder_factory.hpp"
 
-#include <caf/downstream.hpp>
-#include <caf/event_based_actor.hpp>
-#include <caf/expected.hpp>
-#include <caf/io/broker.hpp>
-#include <caf/stateful_actor.hpp>
-#include <caf/stream_source.hpp>
-#include <caf/streambuf.hpp>
-
-#include <unordered_map>
+#include "caf/event_based_actor.hpp"
+#include "caf/io/broker.hpp"
```

Comment:
It builds fine without `event_based_actor`.
```suggestion
#include <caf/io/broker.hpp>
```